src/Command/Environment/EnvironmentInfoCommand.php

```diff
@@ -92,6 +92,10 @@ protected function listProperties(Environment $environment)
             $headings[] = new AdaptiveTableCell($key, ['wrap' => false]);
             $values[] = $this->formatter->format($value, $key);
         }
+
+        $headings[] = 'deployment_type';
+        $values[] = $environment->getSettings()->enable_manual_deployments ? 'manual' : 'automatic';
```

Comment:
```suggestion
        $values[] = $environment->getSettings()->getProperty('enable_manual_deployments', false) ? 'manual' : 'automatic';
```

... because it seems the property doesn't exist everywhere - it's only in git 32?